pkg/operator/client/deploy.go

```diff
@@ -521,7 +518,7 @@ func (c *Client) uninstallWithHelm(v1Beta1ChartsDir, v1Beta2ChartsDir string, ko
 		}
 
 		logger.Infof("running helm with arguments %v", args)
-		cmd := exec.Command(fmt.Sprintf("helm%s", version), args...)
```

Comment:
is this because v3 is the default and you just want to use the default?

Reply:
it's because we removed the helm 2 binary a long time ago but didn't refactor this code.